src/api/crypto/blockchain/BalanceOracle.cpp

```diff
@@ -214,50 +214,37 @@ auto BalanceOracle::Imp::process_registration(Message&& in) noexcept -> void
     const auto haveNym = (2 < body.size());
     auto output = opentxs::blockchain::Balance{};
     const auto& chainFrame = body.at(1);
-    const auto nym = [&] {
-        if (haveNym) {
-
-            return api_.Factory().NymID(body.at(2));
-        } else {
+    const auto nym = api_.Factory().NymID(body.at(2));
```

Comment:
what happen with else? Was it a redundant code?

Reply:
If you take a look at code, nym was never used in context where !haveNym